ONSdigital/eq-survey-runner
# EQ-431 Conditional Display

### What is the context of this PR?

Complete rework of the template preprocessing and the implementation of conditional display.

To achieve this we've implemented a new skip_condition in the schema. If this condition is met then the question is not display. The condition can be either equals or not equals.
### How to review
1. Check out this branch
2. Run the star wars survey
3. If you answer "Bothans" to "Who stole the plans to the 2nd death star" then an additional question appears on the next page. This is a "not equals" check and a non-mandatory question.
4. If you answer "I love them all" to "What do you think of the prequel series" then an additional question appears on the last page. This is a "equals" check and a mandatory question.
### Who worked on the PR

@warrenbailey
